pkgs/build-support/docker/default.nix

```
@@ -679,14 +692,38 @@ rec {
                   extraCommands;
         };
       result = runCommand "docker-image-${baseName}.tar.gz" {
+        nativeBuildInputs = [ pigz coreutils ];
+        passthru.buildArgs = args;
+        passthru.layer = layer;
+        passthru.raw = result_raw;
+        passthru.contentsList = let
+          contentsList = if builtins.isList contents then contents else [ contents ];
```

the lib has a function that does the same thing:
```suggestion
          contentsList = lib.toList contents;
```